dipy/align/tests/test_metrics.py

```diff
@@ -15,6 +16,21 @@ def test_exceptions():
     assert_raises(ValueError, SSDMetric, 3, step_type='unknown_metric_name')
     assert_raises(ValueError, EMMetric, 3, step_type='unknown_metric_name')
 
+    def check_image_size_error(metric):
+        metric.set_static_image(np.arange(np.prod(shape)).reshape(shape),
+                                np.eye(4), np.ones(3), np.eye(3))
+        metric.set_moving_image(np.arange(np.prod(shape)).reshape(shape),
+                                np.eye(4), np.ones(3), np.eye(3))
+        assert_raises(IOError, metric.initialize_iteration)
+
+    metric = CCMetric(2)
+    for shape in itertools.product((5, 8), (8, 5)):
```

Comment:
it will be clearer to others in the future if you add a small comment:
```
# expected to fail for any dimension < 2*radius + 1.
```
and then explicitly set the `radius` argument to `CCMetric`.

For completeness, you could also check that it does not raise when `size == 2*radius + 1`.

Reply:
Agree, done in my last commit